spring-cloud-contract-tools/spring-cloud-contract-spec-pact/pom.xml

```diff
@@ -35,8 +35,7 @@
 		</dependency>
 		<dependency>
 			<groupId>au.com.dius</groupId>
-			<artifactId>pact-jvm-model</artifactId>
-			<optional>true</optional>
+			<artifactId>pact-jvm-consumer-java8_2.12</artifactId>
```

Comment:
Why not optional? Do we want to force that particular version?

Reply:
My bad, that wasn't supposed to happen. Then again, I won't know if it will work with future versions.
(except when somebody files an issue)